cloud_testing/kotlin_poc/build.gradle.kts

```diff
@@ -82,6 +82,7 @@ dependencies {
     // https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.google.apis%22%20AND%20a%3A%22google-api-services-testing%22
     // compile("com.google.apis:google-api-services-testing:v1-rev30-1.23.0")
```

Comment:
since you are going to master, remove this?

Reply:
I'd like to keep the comment because we will be switching back to the official testing jar in 3 months. Currently we have to generate a local testing jar due to the use of private iOS APIs.